src/topology/metric_space/gromov_hausdorff.lean

```diff
@@ -615,8 +615,8 @@ begin
   have : ∀p:GH_space, ∀t:set (p.rep), finite t → ∃n:ℕ, ∃e:equiv t (fin n), true,
   { assume p t ht,
     letI : fintype t := finite.fintype ht,
-    rcases fintype.exists_equiv_fin t with ⟨n, hn⟩,
-    rcases hn with ⟨e⟩,
+    let n := fintype.card t,
+    obtain ⟨e⟩ := fintype.trunc_equiv_fin t,
     exact ⟨n, e, trivial⟩ },
```

Comment:
I think I previously argued for the current style because it avoids needlessly invoking the axiom of choice. Perhaps mathlib doesn't care about that.

Reply:
Yes, basically the question is whether we want to keep or revert commit de3ac17c2ac687da33cfb62f0ad24c7719120efe (and I guess golf a couple of proofs after we revert it).

Reply:
My opinion is that I don't care at all about the axiom of choice, so I'd favor the shorter and more readable version. But I understand that some people may think differently. There is maybe a difference between basic files (where you can hope to avoid it) and more advanced areas where it doesn't make sense any more since many definitions/proofs already rely on it.